README.md

```diff
@@ -20,7 +20,7 @@ specific committee structure information.
 | ---- | ------- | ----------- | ----------- |
 | Christoph Blecker | **[@cblecker](https://github.com/cblecker)** | Red Hat | 2y |
 | Stephen Augustus | **[@justaugustus](https://github.com/justaugustus)** | Cisco | 2y |
-| Paris Pittman | **[@parispittman](https://github.com/parispittman)** | Apple | 2y |
+| Carlos Tadeu Panato Jr. | **[@cpanato](https://github.com/cpanato)**  |Chainguard | 2y |
```

Comment:
Should it not be 1y only?

Reply:
Fixed in https://github.com/kubernetes/steering/pull/261/commits/c0e6eafa131bb5918bd24653233568a44439497f.